actions_helper/commands/create_task_definition.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any
+
+from botocore.client import BaseClient
+
+from actions_helper.commands.get_active_task_definition_by_tag import get_active_task_definition_arn_by_tag
+from actions_helper.outputs import CreateTaskDefinitionOutput
+from actions_helper.utils import set_error
+
+PLACEHOLDER_TEXT = "PLACEHOLDER"
+
+
+def describe_task_definition(ecs_client: BaseClient, task_definition_arn: str, image_uri: str):
+    task_definition = ecs_client.describe_task_definition(taskDefinition=task_definition_arn)["taskDefinition"]
+
+    all_container_images = [
+        container_definition["image"] for container_definition in task_definition["containerDefinitions"]
+    ]
+
+    if len(set(all_container_images)) != 1 or all_container_images[0] != PLACEHOLDER_TEXT:
+        set_error("Not all values for containerDefinitions 'image' equal to 'PLACEHOLDER'")
+
+    for container_definition in task_definition["containerDefinitions"]:
+        container_definition["image"] = image_uri
+
+    del task_definition["taskDefinitionArn"]
+    del task_definition["revision"]
+    del task_definition["status"]
+    del task_definition["requiresAttributes"]
+    del task_definition["compatibilities"]
+    del task_definition["registeredAt"]
+    del task_definition["registeredBy"]
```

Comment:
You can move the keys to a constant above and loop here
```suggestion
    for key in KEYS_TO_DELETE_FROM_TASK_DEFINITION:
        del task_definition[key]
```